scalding-serialization/src/main/scala/com/twitter/scalding/serialization/Serialization2.scala

```diff
@@ -58,6 +58,16 @@ class Serialization2[A, B](val serA: Serialization[A], val serB: Serialization[B
   } yield a + b
 }
 
+object OrderedSerialization2 {
+  def maybeOrderedSerialization[A, B](implicit ordA: Ordering[A], ordB: Ordering[B]): Ordering[(A, B)] = {
```

Comment:
can we actually rename this `maybeOrderedSerialization2`. Sorry, in scala you can't have implicits with colliding names (but you can have normal methods with colliding names as long as their parameters are different). We might want to add `maybe...[2-22]` at some point in the future.